core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -308,13 +318,14 @@ private boolean putInternal(PageId pageId, byte[] page, CacheScope scope, CacheQ
     }
     if (result == PutResult.BENIGN_RACING) {
       Metrics.PUT_BENIGN_RACING_ERRORS.inc();
-    } else if (result == PutResult.INSUFFICIENT_SPACE) {
+    } else if (result == PutResult.INSUFFICIENT_SPACE_EVICTED) {
       Metrics.PUT_INSUFFICIENT_SPACE_ERRORS.inc();
     }
     return false;
   }
 
-  private PutResult putAttempt(PageId pageId, byte[] page, CacheScope scope, CacheQuota quota) {
+  private PutResult putAttempt(PageId pageId, byte[] page, CacheScope scope, CacheQuota quota,
+      boolean forcedToEvict) {
     LOG.debug("putInternal({},{} bytes) enters", pageId, page.length);
```

Comment:
Should we also log `forcedToEvict` or have some metrics around that?

Reply:
@calvinjia  tracked by metrics `PUT_STORE_WRITE_NO_SPACE_ERROR`